base/baseElement.js

```diff
@@ -21,6 +21,10 @@ class BaseElement {
         let EC = protractor.ExpectedConditions;
         await browser.wait(EC.presenceOf(this.protractorElement), ms);
     }
+
+    async getAttribute() {
```

Comment:
1. `getAttribute` should be a generic method, it should wrap protractor method and return attribute which was passed as a method parameter
2. `expect` should be used only in the test body